composer.json

```
@@ -14,9 +14,9 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
+        "php": ">=5.5.9",
         "ajgl/csv": "^0.4",
-        "symfony/framework-bundle": "^2.3"
+        "symfony/framework-bundle": "^2.3|^2.8|^3.4"
```

Symfony 2.7 will be supported until May 2019
```suggestion
        "symfony/framework-bundle": "^2.7|^3.4"
```